internal/attacktechniques/aws/discovery/ec2-get-user-data/main.go

```diff
@@ -0,0 +1,89 @@
+package aws
+
+import (
+	"context"
+	_ "embed"
+	"log"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/config"
+	"github.com/aws/aws-sdk-go-v2/credentials/stscreds"
+	"github.com/aws/aws-sdk-go-v2/service/ec2"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+
+	"github.com/datadog/stratus-red-team/internal/utils"
+	"github.com/datadog/stratus-red-team/pkg/stratus"
+	"github.com/datadog/stratus-red-team/pkg/stratus/mitreattack"
+)
+
+//go:embed main.tf
+var tf []byte
+
+func init() {
+	stratus.GetRegistry().RegisterAttackTechnique(&stratus.AttackTechnique{
+		ID:           "aws.exfiltration.ec2-download-user-data",
```

Comment:
as discussed, we probably want to change it to `discovery`

```suggestion
		ID:           "aws.discovery.ec2-download-user-data",
```